src/mocker.js

```diff
@@ -32,12 +32,12 @@ async function main() {
   }
 }
 
-function saveToSql(vocabulary, cases, events) {
+async function saveToSql(vocabulary, cases, events) {
   deleteFile(`out/${fileNameForCombined()}.sql`);
```

Comment:
I think we should `await` here too.

Reply:
Since  `fs.accessSync` and `fs.unlinkSync` inside deleteFile are synchronous blocking calls, we don't need to specify `await` here.

Reply:
Ok.